src/UniOneClient.php

```diff
@@ -11,110 +11,134 @@
 /**
  * This class is the base class for the UniOne SDK.
  */
-final class UniOneClient {
+final class UniOneClient
+{
+    /**
+     * The HTTP client.
+     *
+     * @var Client
+     */
+    private Client $httpClient;
 
-  /**
-   * The HTTP client.
-   *
-   * @var Client
-   */
-  protected Client $httpClient;
+    /**
+     * UniOne Instance.
+     *
+     * @var string|UriInterface
+     * @todo: Make it configurable.
```

Comment:
I think it's essential to make endpoint configurable in the first alpha release because we would already have it in the Drupal module in the configuration form. We can probably accept it as 2nd argument in the client constructor and have a setter method to be able to change it after initialization.